editors/emacs/Makefile

```diff
@@ -0,0 +1,19 @@
+VERSION = 0.1
+PKG = lambdapi-mode-$(VERSION).tar
+
+$(PKG): $(wildcard *.el)
+	mkdir lambdapi-mode-$(VERSION)
+	cp *.el lambdapi-mode-$(VERSION)
+	tar --create --file $@ lambdapi-mode-$(VERSION)
+	$(RM) -rf lambdapi-mode-$(VERSION)
```

Comment:
Note that instead of all that you can use (up to typo, I have not tried it):
```
tar --create --file $@ --transform 's,^,lambdapi-mode-$(VERSION)/,' $^
```

Reply:
Or even:
```
tar --create --file $@ --transform 's,^,$(patsubst %.tar,%,$@)/,' $^
```